index.html

```diff
@@ -35,7 +39,7 @@ <h2>Renewable electricity autarky</h2>
                 <span class="tooltiptext">Download data</span>
             </a>
             <a class="tooltip" href="https://www.iass-potsdam.de/en/research-group/energy-transition-dynamics" target="_blank">
-                <i class="fas fa-user-circle"></i>
+                <i class="fas fa-user-circle"></i>Europe
```

Comment:
Adding `Europe` here was probably not intended.

Reply:
yes. sorry. was a mistake with search function I assume

Reply:
no problem, just marked it for myself. I am working on the updates now. I will likely close this pull request and pull in changes manually.